packages/builder/src/components/backend/DatasourceNavigator/icons/GoogleSheets.svelte

```diff
@@ -182,3 +182,9 @@
     </g>
   </g>
 </svg>
+
+<style>
```

Comment:
Some of these icons weren't centered, changing this doesn't affect the look of the other places they're consumed negatively.

Reply:
No probs.